source/backend/database/models/recipesModel.js

```diff
@@ -12,4 +12,33 @@ async function getByRecipeId(recipeId) {
   return result;
 }
 
-module.exports = { getByRecipeId };
+/**
+ * create a recipe
+ * @param payload 
+ * @param userId
+ * @returns 
+ */
+async function createRecipe(payload, userId) {
+  payload.userId = userId;
+  const result = await db('recipes')
+    .insert(payload);
+}
+
+/**
+ * remove by user id and recipe id
+ * @param userId
+ * @param recipeId
+ * @return {Promise<void>}
+ */
+ async function deleteRecipe(userId, recipeId) {
```

Comment:
we should probably add a TODO: also remove all recipes with the savedRecipe table with recipeId

Reply:
I forgot what we said in our meeting, but do we still need to remove all recipes from the saved table? I think you said something about joining the tables, so it should be taken care of?

Reply:
Yeah I think we can just leave this for now.